impexp-client/src/main/java/org/citydb/cli/ImpExpCli.java

```diff
@@ -188,6 +194,12 @@ public boolean doKmlExport(String kmlExportFile) throws ImpExpException {
 
 		log.info("Initializing database export...");
 
+		// automatically calculate bounding box --> will overwrite bounding box defined in the config file
+		BoundingBox configBBOX = config.getProject().getKmlExporter().getQuery().getBboxFilter().getExtent();
```

Comment:
Why is the bounding box given in the config file always overriden with a calculated one for KML exports? I don't think that this is a good idea. Maybe there could be an additional command line parameter to control the behaviour of the KML export if required.

Reply:
Right. The bounding box in the config should not always be overriden by a calculated one. 
How about adding an condition check to say: if the bounding box has not defined in the config file, then the config should be automatically filled with a calculated one?

Reply:
@clausnagel Added a CLI flag `-calcBBOX` for auto bounding box calculation (see https://github.com/3dcitydb/importer-exporter/commit/dd90f7c26f41f6078ef36147f35fccc2f8aa4a72)

Reply:
@yaozhihang If a bounding box is not defined in the config file, it shall automatically be calculated from the database (see https://github.com/3dcitydb/importer-exporter/commit/8372709505f045f6d9930b2c16938c245aa1f834)

Reply:
For example the commands:
+ `java -jar lib/impexp-client-4.2.2.jar -shell -kmlExport "export/export.kml" -calcBBOX` 
shall automatically calculate the bounding box regardless of whether a bounding box already exists in the config file or not
+ `java -jar lib/impexp-client-4.2.2.jar -shell -kmlExport "export/export.kml"` shall not override the bounding box in the config file, however if such bounding box does not exist, it shall be automatically calculated from the database